src/integrationTest/java/uk/gov/hmcts/reform/sscs/controllers/CcdCallbackControllerTest.java

```diff
@@ -344,7 +344,7 @@ private ExceptionCaseData exceptionCaseData(Map<String, Object> caseData) {
             VALUE,
             ScannedRecord.builder().docScanDate("2018-10-10")
                 .documentControlNumber("11111")
-                .documentLink(DocumentLink.builder().documentUrl("http://www.bbc.com").build())
+                .documentLink(DocumentLink.builder().documentUrl("http://www.bbc.com").documentBinaryUrl("binaryurl.com").build())
```

Comment:
would be nice to use `http://www.bbc.com/binary` as it is the case always

Reply:
Updated